city_scrapers/spiders/tulok_bocc.py

```
@@ -0,0 +1,150 @@
+    def _parse_links(self, raw_event):
+        """Parse or generate links."""
+        event_id = raw_event.get("id")
+        links = []
+        for f in raw_event.get("publishedFiles", []):
+            file_id = f.get("fileId")
+            if not file_id or not event_id:
+                continue
+            links.append(
+                {
+                    "title": f.get("type") or "Document",
+                    "href": f"{self.portal_base_url}/event/{event_id}/files/agenda/{file_id}",  # noqa
```

Ruff (0.14.8)

Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)